<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Checkout</string>
  </property>
  <widget class="QLabel" name="imglabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>331</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="infolabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>-10</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Қош келдіңіз,өтініш,жүйеге тіркеліңіз</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="nextbtn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>80</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Қосу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="endbtn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>130</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Аяқтау</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameinfolabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Аты:</string>
   </property>
  </widget>
  <widget class="QLabel" name="pnamelabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="priceinfolabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Бағасы</string>
   </property>
  </widget>
  <widget class="QLabel" name="pricelabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="amountinfolabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Саны</string>
   </property>
  </widget>
  <widget class="QLabel" name="overallpriceinfolabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Толық бағасы</string>
   </property>
  </widget>
  <widget class="QLabel" name="overallpricelabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0 тг</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdN">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="controlbtn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>280</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="typelabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>55</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>КГ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="controlbtn_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>330</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="datalabel_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>55</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="datalabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>86</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="datalabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>761</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>761</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Өнім</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Саны</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Бағасы</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>710</y>
     <width>761</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,5">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Барлығы:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="totallabel">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>0000 </string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>тг</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="controlbtn_3">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="balancelbl">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>40</y>
     <width>95</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>0</y>
     <width>95</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Баланс:</string>
   </property>
  </widget>
  <widget class="QLabel" name="infolabel_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="exitbtn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>180</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Шығу</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
